docs/advanced-topics/executor-service.md

```diff
@@ -27,21 +36,21 @@ use any of the configuration options provided in `ExecutorServiceConfiguration`
 convenient to use the provided configurations. The configuration options provided in `ExecutorServiceConfiguration` 
 are typically those configuration options which are provided via configurations from `storage.parallel-backend-executor-service`.
 
-# Cassandra backend ExecutorService
+## Cassandra backend ExecutorService
 
-Apart from the general executor service mentioned in the previous section, Cassandra backend uses an additional 
-`ExecutorService` to process CQL queries by default (see `storage.cql.executor-service.enabled` configuration option).  
+Cassandra backend has multi-key queries support. Thus, `storage.parallel-backend-executor-service` is ignored and not 
+in use when Cassandra backend is in use.  
+Nevertheless, Cassandra backend doesn't use `storage.parallel-backend-executor-service` for IO operations it has its 
```

Comment:
This sentence seems incomplete. Do you mean:

> Although Cassandra backend doesn't use `storage.parallel-backend-executor-service`, it ...

Reply:
I tried to rephrase this sentence here as you suggested. Let me know that you think about it.

Reply:
Looks good 👍